src/components/menu/menu/menu.component.ts

```diff
@@ -241,6 +242,10 @@ export class MenuComponent implements AfterContentInit, OnDestroy, OnChanges {
 
         // emit the keydown event
         this._onKeydown$.next(event);
+
+        if (event.keyCode === TAB) {
```

Comment:
Use the blur event instead of listening for a tab key. Some screen readers use other keys as well as tab to change focus.